### PR TITLE
feat: add reference links for cel-go in footer (#27)

### LINKED ISSUE #27: Add reference links for cel-go and Language definition overview.
## Where?

I'd like to suggest to include two references links:  
    * https://github.com/google/cel-go --> at cel-go version on the footer.
    * https://github.com/google/cel-spec/blob/master/doc/langdef.md  --> create a text "Language definition" on the footer too, next to cel-go.

## Who?
It helps CEL users find  information by consulting the "Language Definition" link, which serves as an reference for learning about CEL.

### PR DESCRIPTION
**Pull request by marcelloale**
## Description
Add reference links for cel-go and Language definition to the footer of the site.

## Linked Issues
Closes #27

### CODE REVIEW

**Comment by lucasjct** [comment]
Thank you for your time, @marcelloale.

We have some considerations about your PR.

We need the links and text, "Language definition" and "cel-go v0.16.0," to be separated. You can use a pipe for this. Please see the example below:

image  

Let me know if you have any question.
Thanks!

**Comment by marcelloale** [comment]
Thank you for your feedback, @lucasjct . I have made the necessary adjustments in css to the PR.

File: web/index.html [hunk 1]
```diff
@@ -139,8 +139,11 @@
 		<span>Powered by
 			<a href="https://www.getup.io/en/getup-open-source/" target="_blank">Getup</a>
 		</span>
+		<span class="langdef">
+			<a href="https://github.com/google/cel-spec/blob/master/doc/langdef.md" target="_blank">Language Definition</a> |
+		</span>
 		<span class="version">
-			cel-go <span id="version">...</span>
+			<a href="https://github.com/google/cel-go" target="_blank">cel-go</a> <span id="version">...</span>
```
Comment (lucasjct) [inline web/index.html:146]:
```suggestion
			<a href="https://github.com/google/cel-go" target="_blank">cel-go <span id="version">...</a></span>
```

**Comment by marcelloale** [comment]
Done ✅

**Comment by lucasjct** [comment]
@marcelloale, thank you for your collaboration! 

Your PR has been accepted and will be merged.

**Pull request merged**